src/Host/Client/Impl/Definitions/IRSessionCallback.cs

```diff
@@ -35,7 +36,18 @@ public interface IRSessionCallback {
         /// Set locator mode in the plot window
```

Comment:
If we want to refactor things here, the right approach would be to keep only the core callbacks that the host itself requires (ReadConsole etc), and add an extensible callback registration mechanism for everything else. But it's a relatively big change, so maybe not at this point.